Classes/Parser/Expression/NodeList.php

```diff
@@ -34,7 +34,7 @@ public static function parse(Lexer $lexer): array
         while (!$lexer->isEnd()) {
             if ($lexer->isOpeningBracket()) {
                 $lexer->consume();
-                if ($currentText) {
+                if ($currentText !== '') {
```

Comment:
should we think about `trim()` here?

Reply:
What about https://github.com/neos/fusion-afx/pull/31/files#diff-5c2ca3faa504ec000342c9d5490af938R85?

Reply:
@gerhard-boden what would trim() achieve here? Empty strings in the payload are handled in the afx-service and not in the payload. The service then decides on which whitespace is meaningful or not.

I agree that the second comparison below should be adjusted aswell.

Reply:
thanks for clarifying! Just saw the condition for the empty string and figured it could be possible that the payload contains whitespace, but as you pointed it's not possible and therefore nothing to worry about. Please ping me whenever the second condition is handled for my approval vote :+1: